library/src/main/java/com/datatorrent/lib/io/block/BlockReader.java

```diff
@@ -1,66 +0,0 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package com.datatorrent.lib.io.block;
-
-import java.io.IOException;
-import java.net.URI;
-
-import org.apache.hadoop.fs.FileSystem;
-
-import com.datatorrent.api.AutoMetric;
-
-/**
- * BlockReader extends {@link FSSliceReader} to accept case insensitive uri
- */
-public class BlockReader extends FSSliceReader
-{
-  @AutoMetric
```

Comment:
The metrics wasn't doing anything in this class. If was just being initialized to 0 in begin window. Wasn't getting updated